auth/verifier.go

```diff
@@ -35,15 +35,15 @@ func (verifier *PermissionsVerifier) CheckAuthorisation(ctx context.Context, act
 	}
 
 	if len(missingPermissions) > 0 {
-		log.Event(ctx, "action forbidden caller does not process the required permissions", log.Data{
+		log.Info(ctx, "action forbidden caller does not process the required permissions", log.Data{
```

Comment:
Wasn't sure if this should be a log.Warn or a log.Info?

Reply:
We haven't traditionally uses log levels that well. All out events tend to be info or error which makes filtering logs in kibana more difficult as you can't reduce the search space as much. 

I think I'd personally go with warning here as _something wasn't right_ but not an error with the service.

Reply:
That makes sense, will change

Reply:
changed to log.Warn